package.json

```diff
@@ -18,24 +18,30 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.11.0",
```

Comment:
If you are going to add this package it would make sense to add it to the technologies in the readme as well.